templates_jinja2/team_partials/team_info.html

```diff
@@ -15,6 +15,7 @@
       {% if team.location %}<span class="glyphicon glyphicon-map-marker"></span> From <a href="http://maps.google.com/maps?q={{ team.location|urlencode }}" target="_blank">{{ team.location }}</a><br>{% endif %}
       {% if team.name %}<span class="glyphicon glyphicon-info-sign"></span> aka <em>{{ team.name }}</em><br>{% endif %}
       {% if team.rookie_year %}<span class="glyphicon glyphicon-calendar"></span> First competed in {{ team.rookie_year }}<br>{% endif %}
+      {% if last_competed and year != current_year %}Last competed in {{ last_competed }}.{% endif %}
```

Comment:
Missing a \<br\> tag. This causes the "last competed in" stat and the team website to be on the same line.
There's also a period that looks out of place.
![2016-08-06 05_38_33-d penguineers - team 1717 2015 - the blue alliance](https://cloud.githubusercontent.com/assets/3937393/17456282/12395894-5b98-11e6-8f74-440d2141f618.png)